### PR TITLE
#165812170 Integrate code climate to GitHub

### PR DESCRIPTION
**Pull request by a-braham**
#### What does this PR do?
This pull request integrates code climate to GitHub so developers can view the quality of code and maintainability. 

#### Description of Task to be completed?
- Ensure that the code is formatted to PEP8 standards

#### How should this be manually tested?
- change branch to ch-code-climate-165812170 in GitHub and click on the `maintainability` badge to open code climate.

#### What are the relevant pivotal tracker stories?
- [165812170](https://www.pivotaltracker.com/story/show/165812170)

#### Screenshots
Screenshot 2019-05-06 at 19 02 40

### CODE REVIEW

**Review by Ogutu-Brian — APPROVED** [review]
LGTM

**Comment by Bakley** [comment]
Screenshot 2019-05-06 at 15 47 38
Good work on this @a-braham

**Review by Bakley — APPROVED** [review]

**Review by matthenge — APPROVED** [review]
LGTM

File: .codeclimate.yml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+version: "2"
+plugins:
+  pep8:
+    enabled: true
+exclude_patterns:
+  - "**/tests/"
+  - "**/migrations/"
+  - "**/tests.py"
```
Comment (3N61N33R) [inline .codeclimate.yml:8]: Hey @a-braham 
Add a new line on this file

**Pull request merged**